### ISSUE TITLE
COMPONENTS/Command: Backspace bug

### ISSUE THREAD

**Issue by brandonmcconnell** [shadcn-ui/ui#1052]
Once items are filtered from the [Command menu](https://ui.shadcn.com/docs/components/command), they do not reappear until the Command menu's input field is completely cleared

https://github.com/shadcn-ui/ui/assets/5913254/9e739f1e-2941-4618-a86d-ce412401b2ba

**Comment by ilyichv** [comment]
`CommandItem` is missing `value` prop. If you want I can fix with a quick PR.

**Comment by divyammadhok** [comment]
@brandonmcconnell @ilyichv I also faced the same issue and noticed that an issue has already been created. I went ahead and raised a PR for the same so that the users browsing the docs don't face the same issue

PR: https://github.com/shadcn-ui/ui/pull/1286

**Comment by miquelvir** [comment]
Duplicate of #1450 - already fixed together with others in #1522 

Please close as duplicate

**Comment by brandonmcconnell** [comment]
@miquelvir @shadcn This is still not fixed. I just checked the latest version of the page I mentioned in the original issue brief, and the same issue persists.

**Comment by miquelvir** [comment]
@brandonmcconnell yes, the PR is still open (has not been merged). I meant that when the PR is merged then this should be fixed.

**Comment by brandonmcconnell** [comment]
@miquelvir Ah got it. Thanks. We should probably call those duplicates out I try e PR in GitHub fashion like…

Fixes #9998
Fixes #9999

I'll check the issue again once that PR gets merged. Thanks for calling that out!

**Comment by Nithur-M** [comment]
I am still facing this issue but the component is working properly on the official site? how is that possible?

**Comment by shadcn** [comment]
This issue has been automatically marked as stale due to one year of inactivity. It will be closed in 7 days unless there’s further input. If you believe this issue is still relevant, please leave a comment or provide updated details. Thank you. (This is an automated message)

**Comment by brandonmcconnell** [comment]
This was already fixed together with others in #1522

Please close as duplicate of #1450
